packages/rn-tester-e2e/jest.setup.js

```diff
@@ -0,0 +1,31 @@
+const wdio = require('webdriverio');
+import capabilities from './e2e-config.js';
+import { beforeEach, afterEach, jest } from '@jest/globals';
+
+
+jest.setTimeout(40000);
+jest.retryTimes(3);
+
+let driver;
+const config = {
+  path: '/wd/hub',
+  host: 'localhost',
+  port: 4723,
+  waitforTimeout: 30000,
+  logLevel: 'silent',
```

Comment:
I think we might want it to be at least error:
```suggestion
  logLevel: 'error',
```

[See here ](https://github.com/facebook/react-native/pull/35824#discussion_r1071313061)

Reply:
done